quickstarts/microsoft.kubernetes/fleet/private-hubful-fleet-deploy.json

```
@@ -0,0 +1,94 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vnet_name": {
+            "type": "String"
+        },
+        "fleets_priv_fleet_name": {
+            "type": "String"
+        },
+        "fleets_sp_object_id": {
+            "type": "String"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Network/virtualNetworks",
+            "apiVersion": "2024-05-01",
+            "name": "[parameters('vnet_name')]",
+            "location": "eastus",
```

think you can write this to avoid hard coding

```suggestion
            "location": "[resourceGroup().location]",
```